src/main/java/net/sf/jabref/importer/fetcher/IEEEXploreFetcher.java

```diff
@@ -771,19 +642,20 @@ private int getNumberOfHits(String page, String marker, Pattern pattern) throws
      */
     private String getResults(URL source) throws IOException {
 
-        InputStream in = source.openStream();
-        StringBuilder sb = new StringBuilder();
-        byte[] buffer = new byte[256];
-        while (true) {
-            int bytesRead = in.read(buffer);
-            if (bytesRead == -1) {
-                break;
-            }
-            for (int i = 0; i < bytesRead; i++) {
-                sb.append((char) buffer[i]);
+        try(InputStream in = source.openStream()) {
+            StringBuilder sb = new StringBuilder();
```

Comment:
This block: reading a string from an Input stream is done also just below in the code in the same way. Hence, it is duplicated code. 

I would suggest that a util method is created for converting an InputStream to a String. Normally, a charset must be set when converting bytes to chars. I would think that IEEExplore always returns utf-8, hence, I would use that. Please see http://www.baeldung.com/convert-input-stream-to-string